### PR TITLE
docs: generate release notes on build

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
Notes: generate release notes as pdf on build

### CODE REVIEW

**Review by VishwasShashidhar — APPROVED** [review]

**Comment by VishwasShashidhar** [comment]
@johankwarnmarksymphony It looks good! Approved.

One suggestion though, it'll be nice to have links to the ticket numbers in the Release Notes so people can directly open it without having to copy paste anything.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+-------------------------------------------------------------------
+## SDA 9.2.x
+
+### 9.2.0
+
+1. SDA-2402 CLIENT Annotate screen capture - PHASE 1
+2. SDA-2404 CLIENT Toast Action: Phase 1 
+3. SDA-2385 CORE Windows Wix installer
```
Comment (VishwasShashidhar) [inline RELEASE_NOTES.md:11]:
```suggestion
1. [SDA-2402 CLIENT Annotate screen capture - PHASE 1](https://perzoinc.atlassian.net/browse/SDA-2402)
2. [SDA-2404 CLIENT Toast Action: Phase 1](https://perzoinc.atlassian.net/browse/SDA-2404)
3. [SDA-2385 CORE Windows Wix installer](https://perzoinc.atlassian.net/browse/SDA-2385)
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+* 9.3.5 (Electron-version)
+* 83    (Chromium-version)
+
+-------------------------------------------------------------------
+## SDA 9.1.x
+
+### SDA 9.1.4
+
+1. SDA-2694 [Point72] blank screen windows-only
+2. SDA-2731 Blank screen after upgrade (fresh installation) windows-only
```
Comment (VishwasShashidhar) [inline RELEASE_NOTES.md:22]:
```suggestion
1. [SDA-2694 Point72 blank screen windows-only](https://perzoinc.atlassian.net/browse/SDA-2694)
2. [SDA-2731 Blank screen after upgrade (fresh installation) windows-only](https://perzoinc.atlassian.net/browse/SDA-2731)
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+* 9.3.5 (Electron-version)
+* 83    (Chromium-version)
+
+### SDA 9.1.3
+
+1. SDA-2559 CORE Remove Cache on New Install
+2. SDA-2359 CORE Instinet Prod - SDA blank screen 
+3. SDA-2524 CORE Unpreventable top-level navigation
+4. SDA-2525 CORE Context isolation bypass via prevented window.open
```
Comment (VishwasShashidhar) [inline RELEASE_NOTES.md:32]:
```suggestion
1. [SDA-2559 CORE Remove Cache on New Install](https://perzoinc.atlassian.net/browse/SDA-2559)
2. [SDA-2359 CORE Instinet Prod - SDA blank screen](https://perzoinc.atlassian.net/browse/SDA-2359)
3. [SDA-2524 CORE Unpreventable top-level navigation](https://perzoinc.atlassian.net/browse/SDA-2524)
4. [SDA-2525 CORE Context isolation bypass via prevented window.open](https://perzoinc.atlassian.net/browse/SDA-2525)
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+* 9.3.2 (Electron-version)
+* 83    (Chromium-version)
+
+### SDA 9.1.2
+
+1. SDA-2417 Add E2EE encryption flag requested by RTC part2
+2. SDA-2418 CORE Barclays - SDA 6.x - 9.x takes noticeably longer to start than 3.x
+3. SDA-2406 CORE GS Prod: Mac SDA 6.1 polling for python3, pip3 and other packages
+4. SDA-2465 CORE GS - SDA/RTC - Exception when started Screen Share
```
Comment (VishwasShashidhar) [inline RELEASE_NOTES.md:42]:
```suggestion
1. [SDA-2417 Add E2EE encryption flag requested by RTC part2](https://perzoinc.atlassian.net/browse/SDA-2417)
2. [SDA-2418 CORE Barclays - SDA 6.x - 9.x takes noticeably longer to start than 3.x](https://perzoinc.atlassian.net/browse/SDA-2418)
3. [SDA-2406 CORE GS Prod: Mac SDA 6.1 polling for python3, pip3 and other packages](https://perzoinc.atlassian.net/browse/SDA-2406)
4. [SDA-2465 CORE GS - SDA/RTC - Exception when started Screen Share](https://perzoinc.atlassian.net/browse/SDA-2465)
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+-------------------------------------------------------------------
+## SDA 9.0.x
+
+### SDA 9.0.3
+
+1. SDA-2287 Context Isolation: Missing Hamburger menu when setting contextIsolation = false
+2. SDA-2285 Memory Refresh: Update electron API to support memory refresh in 9.x version
+3. RTC-8098 RTC/SDA:  "Meeting failed" to many meetings from SDA 9.0.x
+4. SDA-2229 [WFC] Unable to login to Symphony on SDA but able to login using Chrome on a macOS
```
Comment (VishwasShashidhar) [inline RELEASE_NOTES.md:55]:
```suggestion
1. [SDA-2287 Context Isolation: Missing Hamburger menu when setting contextIsolation = false](https://perzoinc.atlassian.net/browse/SDA-2287)
2. [SDA-2285 Memory Refresh: Update electron API to support memory refresh in 9.x version](https://perzoinc.atlassian.net/browse/SDA-2285)
3. [RTC-8098 RTC/SDA:  "Meeting failed" to many meetings from SDA 9.0.x](https://perzoinc.atlassian.net/browse/RTC-8098)
4. [SDA-2229 [WFC] Unable to login to Symphony on SDA but able to login using Chrome on a macOS](https://perzoinc.atlassian.net/browse/SDA-2229)
```

**Review by psjostrom — APPROVED** [review]

**Pull request merged**